packages/gatsby-dev-cli/README.md

```diff
@@ -35,6 +31,8 @@ this program running.
 Typically you'll also want to run `npm run watch` in the Gatsby repo to set up
 watchers to build Gatsby source code.
 
+More detailed instruction for setting up your Gatsby repository are found [here](/CONTRIBUTING.md).
```

Comment:
How about you just link to https://www.gatsbyjs.org/docs/how-to-contribute/ (which is just CONTRIBUTING.md)